Source/JavaScriptCore/ftl/FTLLowerDFGToB3.cpp

```diff
@@ -1200,7 +1200,11 @@ class LowerDFGToB3 {
             compileGetPrototypeOf();
             break;
         case GetWebAssemblyInstanceExports:
+#if ENABLE(WEBASSEMBLY)
             compileGetWebAssemblyInstanceExports();
+#else
+            RELEASE_ASSERT_NOT_REACHED();
```

Comment:
unreachable_for_platform?

Reply:
What exactly is your question here? But about how..., I'm building the GTK port, which has public options to disable WASM, so it's a custom target. But if you're hint is about the code then I need more info.

Reply:
`UNREACHABLE_FOR_PLATFORM` is (effectively) an alias for `RELEASE_ASSERT_NOT_REACHED` that may be more appropriate here

Reply:
Ah, ok, I will change it. Copied from other switch cases. Thx for the info.